src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -8,15 +8,48 @@
 using System;
 using System.Device.Gpio;
 using System.Runtime.InteropServices;
+#if NET6_0_OR_GREATER
+using System.Runtime.Loader;
+using System.Reflection;
+#endif
 // Since this is only used on Linux, and in C on Linux sizeof(long) == sizeof(void*) this is a valid alias.
 using NativeLong = System.IntPtr;
 
 internal partial class Interop
 {
-    internal partial class libgpiod
+    internal static partial class Libgpiod
     {
+#if NET6_0_OR_GREATER
+        private const string LibgpiodLibrary = "libgpiod.so.2";
+#else
         private const string LibgpiodLibrary = "libgpiod";
-        internal static IntPtr InvalidHandleValue = new IntPtr(-1);
+#endif
+        internal static IntPtr InvalidHandleValue;
+
+        static Libgpiod()
+        {
+            InvalidHandleValue = new IntPtr(-1);
+#if NET6_0_OR_GREATER
```

Comment:
just to double check; with any other version, we will have the existing behavior, right? In that case, looks all good to me!

Reply:
The following line is what always existed in current versions:
```
        private const string LibgpiodLibrary = "libgpiod";
```

This line of code will now used only for `netstandard` (which we basically use only for Mono on Linux).